### PR TITLE
Change now-playing embed to display time in brackets for consistency

### PR DESCRIPTION
**Pull request by ncovercash**
When displaying the queue, the times of each track are displayed in brackets.  This does this in both the now playing embed and selectSuccess strings.

### CODE REVIEW

File: FredBoat/src/main/java/fredboat/command/admin/AnnounceCommand.java [hunk 1]
```diff
@@ -129,7 +129,7 @@ public void onInvoke(@Nonnull CommandContext context) {
 
     private static void printProgress(Message message, int done, int total, int error) {
         CentralMessaging.editMessage(message, MessageFormat.format(
-                            "[{0}/{1}]{2,choice,0#|0< {2} failed}",
+                            "`[{0}/{1}]`{2,choice,0#|0< {2} failed}",
```
Comment (napstr) [inline FredBoat/src/main/java/fredboat/command/admin/AnnounceCommand.java]: Insignificant formatting changes to bot admin internal command are not necessary.

**Comment by smileytechguy** [comment]
My bad, I didn't see that bit about Crowdin.  I'll revert the above commits and apply it to the correct now playing command, I did it in the wrong file.

File: FredBoat/src/main/java/fredboat/command/admin/AnnounceCommand.java (hunk 1 above)
Comment (smileytechguy) [reply FredBoat/src/main/java/fredboat/command/admin/AnnounceCommand.java]: See reversions and new changes.

**Comment by ncovercash** [comment]
Hey – was looking at my old PRs and was wondering if I had resolved the issues in the review (or if the bug was still present)?

**Comment by Frederikam** [comment]
Hi

It looks like your PR was simply forgotten about two years ago. Sorry about that. It's no longer possible to merge.

The new file is here:
https://github.com/Frederikam/FredBoat/blob/dev/FredBoat/src/main/java/fredboat/command/music/info/NowplayingCommand.kt#L104-L105

**Comment by ncovercash** [comment]
No worries – glad it was fixed!

**Pull request closed without merging**